flink-table-store-core/src/main/java/org/apache/flink/table/store/CoreOptions.java

```diff
@@ -147,6 +147,12 @@ public class CoreOptions implements Serializable {
                     .defaultValue(WriteMode.CHANGE_LOG)
                     .withDescription("Specify the write mode for table.");
 
+    public static final ConfigOption<Boolean> WRITE_SKIP_COMPACTION =
+            ConfigOptions.key("write.skip-compaction")
```

Comment:
Is it better to replace `write.skip-compaction` with `compaction.skip`?

Reply:
I think `write.compaction-skip` is OK.